ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -534,10 +534,13 @@ dateTimeFunctionName
     | DAY_OF_MONTH
     | DAY_OF_WEEK
     | DAY_OF_YEAR
+    | EXTRACT
```

Comment:
Please, copy to PPL full implementation of `extract` from #221. And for `get_format` function.

Reply:
Thank you for your help with this, updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/283/commits/1c64d5c6400e6de10759d9934ad0117e1c8123a4